pkg/controller/testrun_control_update.go

```diff
@@ -104,6 +108,17 @@ func updateStepsStatus(tr *tmv1beta1.Testrun, wf *argov1.Workflow) {
 				continue
 			}
 
+			if strings.Contains(argoNodeStatus.Message, argoctrl.ErrDeadlineExceeded.Error()) {
+				completionTime := metav1.NewTime(stepStatus.StartTime.Add(time.Duration(*stepStatus.TestDefinition.ActiveDeadlineSeconds) * time.Second))
```

Comment:
Maybe decouple a little bit for better reading?
```suggestion
				testDurationMs := time.Duration(*stepStatus.TestDefinition.ActiveDeadlineSeconds) * time.Second
				completionTime := metav1.NewTime(stepStatus.StartTime.Add(testDurationMs))
```

Reply:
done